plugins/katello/3.15/user_guide/remote_databases/index.md

```diff
@@ -24,6 +24,8 @@ There are two ways to deploy Foreman with remote databases:
     * restore the DBs on remote servers
     * run foreman-installer with right parameters pointing to the databases. It re-configures the databases and start all the services with new DB locations
 
+### Important note: Katello 3.15 changes the default database for Pulp from PostgreSQL v9.x to PostgreSQL v12. External databases should be upgraded to avoid conflicts.
```

Comment:
One thing that a lot of 3.15 users ran into was that Pulpcore doesn't support external databases in this version, so you'll always end up with a local database. IMHO it was a mistake to release Pulpcore without external DBs and it would have saved a lot of support questions if we at least documented this.

Reply:
In that case maybe we should remove "External databases..." and instead point out that 3.15 doesn't support external databases?

Reply:
You can still use Foreman and Candlepin external databases. The Foreman DB must be updated to at least 10 on Foreman 2.0.